InnerEye/Scripts/move_model.py

```diff
@@ -0,0 +1,127 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import sys
+from pathlib import Path
+from typing import Tuple
+
+import json
+import param
+from attr import dataclass
+from azureml.core import Environment, Model, Workspace
+
+from InnerEye.Azure.azure_config import AzureConfig
+from InnerEye.Common import fixed_paths
+
+print(f"Starting runner at {sys.argv[0]}")
+innereye_root = Path(__file__).absolute().parent.parent.parent
+if (innereye_root / "InnerEye").is_dir():
+    innereye_root_str = str(innereye_root)
+    if innereye_root_str not in sys.path:
+        print(f"Adding InnerEye folder to sys.path: {innereye_root_str}")
+        sys.path.insert(0, innereye_root_str)
+
+from InnerEye.Common.generic_parsing import GenericConfig  # noqa: E402
```

Comment:
This sys.path massage is only necessary for things running in AML, you can remove that, and the #noqa in line 25